.gitignore

```diff
@@ -1,6 +1 @@
-# Ignore .DS_Store files on Mac
-.DS_Store
-
-# Files that might appear on external disks
-.Spotlight-V100
-.Trashes
+/node_modules/*
```

Comment:
@davidlimcheng let's change this to `node_modules/` instead.
We also don't want directories `node_modules` lower than the root to be checked in.